src/index.js

```diff
@@ -42,21 +42,21 @@ async function start(fields) {
 async function handle2FA(fields) {
   if (fields.error) {
```

Comment:
Maybe doing a `const boundTwoFACodeAttempts = twoFACodeAttempts.bind(this)` above will be better to avoid repetition here?

Reply:
It will be removed by some current changes made in konnector-libs, since it's temporary let's merge it.

Reply:
You are right. But this code will disappear with the implementation in cozy-konnector-libs. Then let's move on.